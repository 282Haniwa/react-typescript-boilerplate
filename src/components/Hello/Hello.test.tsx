import * as React from 'react';
import { shallow } from 'enzyme';
import { Hello } from './Hello';

describe('Hello.tsx', () => {
  test('Hello component test', () => {
    const hello = shallow(<Hello compiler="TypeScript" framework="React" />);

    // Interaction demo
    expect(hello.text()).toEqual('Hello from TypeScript and React!');

    // Snapshot demo
    expect(hello).toMatchSnapshot();
  });
});
