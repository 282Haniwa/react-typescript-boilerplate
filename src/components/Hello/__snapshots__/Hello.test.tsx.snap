// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello component test 1`] = `
<h1>
  Hello from 
  TypeScript
   and 
  React
  !
</h1>
`;
